test/minting2/MintControllerTests.js

```diff
@@ -138,6 +138,43 @@ async function run_tests(newToken, accounts) {
         // increment minter allowance
         await expectRevert(mintController.incrementMinterAllowance(amount, {from: Accounts.controller1Account}));
    });
+
+   it('decrement minter allowance', async function () {
+        // configure controller & minter
+        var amount = 500;
+        await mintController.configureController(Accounts.controller1Account, Accounts.minterAccount, {from: Accounts.mintOwnerAccount});
+        await mintController.configureMinter(amount, {from: Accounts.controller1Account});
+        expectedMintControllerState.controllers['controller1Account'] = Accounts.minterAccount;
+        expectedTokenState.push(
+            { 'variable': 'isAccountMinter.minterAccount', 'expectedValue': true },
+            { 'variable': 'minterAllowance.minterAccount', 'expectedValue': new BigNumber(amount) },
+        );
+        await checkMINTp0([token, mintController], [expectedTokenState, expectedMintControllerState]);
+
+        // decrement minter allowance
+        await mintController.decrementMinterAllowance(amount, {from: Accounts.controller1Account});
+        expectedTokenState = [
+            { 'variable': 'masterMinter', 'expectedValue': mintController.address },
+            { 'variable': 'isAccountMinter.minterAccount', 'expectedValue': true },
+            { 'variable': 'minterAllowance.minterAccount', 'expectedValue': bigZero },
+        ];
+        await checkMINTp0([token, mintController], [expectedTokenState, expectedMintControllerState]);
+   });
+
+   it('only controller decrements allowance', async function () {
+        await expectRevert(mintController.decrementMinterAllowance(0, {from: Accounts.controller1Account}));
+   });
```

Comment:
use `expectError` to check for the correct error message.

Reply:
done